### DISCUSSION TITLE
Generic .env file suggestion

### DISCUSSION THREAD

**Discussion by lbdyck** [zopencommunity/meta#318] [General]
Can the .env be updated to exclude the directory version info when creating the environment variables:

![image](https://github.com/ZOSOpenTools/meta/assets/42328411/4cb85d57-ae0a-4588-9110-2e2558606331)

RIght now it is using the $PWD which is fine but since that is the full directory name (e.g. `git-2.41.0`) which was accessed from the symlink of `git` the version is there. This would reduce the size of the environment variables and simplify creating a profile or environment file to eliminate the need to run the .env files everytime (see `genprofile` at https://github.com/lbdyck/zot_meta/blob/main/tools/genprofile
(a pull request was generated).

**Comment by MikeFultonDev** [comment]
With the move to install into one location, and therefore one PATH and LIBPATH and MANPATH entry, I think this is 'fixed'. Please comment @lbdyck
